centaur/src/main/resources/standardTestCases/write_json/write_json_version_develop.wdl

```diff
@@ -0,0 +1,44 @@
+version development
+
+struct Random {
+    Int index
+    String name
+}
+
+workflow write_json_version_develop {
+    input {
+        Array[Int] indices = [0, 1, 2]
+        Map[String, String] map_input = {"genre": "comedy", "movie": "mr. bean"}
+        Pair[Int, String] pair_input = (1, "abc")
+    }
+    scatter (i in indices) {
+        call create_single_object { input: i=i }
```

Comment:
TOL (not required, just curious): Is there a reason you have to do this as a call rather than 

```wdl
scatter (i in indices) {
  Random randoms = object {index: i, name: "mr_bean"}
}

[...]

output {
        String object_out = read_string(write_json(randoms[0]))
        String array_object_out = read_string(write_json(randoms))
}
```

Reply:
To be honest, initially I was trying to have `task`s in the WDL. But this is better. No tasks, no Papi calls, no VMs. Once this PR is merged I will create another PR with these changes. Thank you!